src/components/ObsEdit/ObsEdit.js

```diff
@@ -49,11 +48,11 @@ const ObsEdit = ( ): Node => {
   useEffect( ( ) => {
     // when opening an observation from ObsDetails, fetch the local
     // observation from realm
-    if ( localObservation ) {
+    if ( localObservation && !currentObservation ) {
```

Comment:
What was this for? I think it's making it so when you view obs detail, edit the obs, back out to My Obs, view another obs, edit that, you end up editing the original obs. In theory you can *only* edit local observations, right? so `localObservation` should always exist and if it does we should make 100% sure that's the current obs in context?

Reply:
I added some more detailed comments here to clarify what this is for. Basically, we need to set the local observation in context only on the initial render, otherwise anytime a user tries to edit the identification, their identification will get overwritten by the local observation being set to context again.

This check means we only set the local observation into context once, and then the id can be edited as expected in ObsEdit.

Reply:
FWIW, I *think* I addressed this in 28f4c2035f0c537ba83bd46ffe1647a28e100e4d